e2e/branch_protection_test.go

```diff
@@ -163,9 +163,9 @@ var _ = Describe("E2E TEST:"+checks.CheckBranchProtection+" (repo rules)", func(
 			expected := scut.TestReturn{
 				Error:         nil,
 				Score:         3,
-				NumberOfWarn:  2,
-				NumberOfInfo:  4,
-				NumberOfDebug: 2,
+				NumberOfWarn:  4,
+				NumberOfInfo:  5,
+				NumberOfDebug: 1,
```

Comment:
overall: these changes are expected. just leaving a comment for discoverability later, can resolve immediately. 

the increase warns is from properly detecting when codeowners + last push approval aren't set with repo rules. This is the bugfix Diogo fixed in `translatePullRequestRepoRule`
the increase in info is from requiring a PR.
the debug thats now gone was from being unable to detect last push approval.